domains/client/domain-operator/src/bundle_processor.rs

```diff
@@ -118,11 +118,6 @@ where
             ))
         })?;
 
-        if local_receipt.invalid_bundles != receipt.invalid_bundles {
```

Comment:
`inboxed_bundles` is checked in the `domain_block_processor` but not the `ReceiptValidator` which is used in the `block-preprocessor` since we don't consider bundle as invalid due to invalid ER anymore. And you remind me that `ReceiptValidator` should be removed completely.